src/zir_sema.zig

```diff
@@ -826,7 +828,112 @@ fn analyzeInstCall(mod: *Module, scope: *Scope, inst: *zir.Inst.Call) InnerError
 
     const ret_type = func.ty.fnReturnType();
 
-    const b = try mod.requireRuntimeBlock(scope, inst.base.src);
+    const b = try mod.requireFunctionBlock(scope, inst.base.src);
+    if (b.is_comptime) {
```

Comment:
Should you change this to make it `if (b.is_comptime or inst.kw_args.modifier == .compile_time)` ?
Or will this come later with the hashing and determining if something can be called at comptime?

Reply:
Yep, good call. I'm actually looking into right now what it would look like to support `inline` function calls too, because I think the problem is related.